tests/test_pex.py

```diff
@@ -34,22 +34,28 @@
     temporary_content,
     write_simple_pex,
 )
+from pex.typing import TYPE_CHECKING
 from pex.util import named_temporary_file
 
 try:
     from unittest import mock
 except ImportError:
     import mock  # type: ignore[no-redef]
 
+if TYPE_CHECKING:
+    from typing import Dict, Iterator, Union
+
 
 def test_pex_uncaught_exceptions():
+    # type: () -> None
     body = "raise Exception('This is an exception')"
     so, rc = run_simple_pex_test(body)
-    assert b"This is an exception" in so, "Standard out was: %s" % so
+    assert b"This is an exception" in so, "Standard out was: %r" % so
```

Comment:
MyPy warns that this will cause you to bring `b"abc"`, with the `b` prefix, when using Py3. It wants us to be explicit about that.